genome_kit/data_manager.py

```diff
@@ -255,8 +255,21 @@ def upload_file(self, filepath: str, filename: str, metadata: Dict[str, str]=Non
 
     @lru_cache
     def list_available_genomes(self):
+        names = set()
+
+        def get_genomes(filenames):
+            return (
+                name.rpartition(".")[0]
+                for name in filenames
+                if name.endswith(".2bit") or name.endswith(".cfg")
+            )
+
+        names.update(get_genomes(os.listdir(self.data_dir)))
```

Comment:
This is a good change (`list_available_genomes` also returns genomes available only in the local dir), can you explicitly add that to the PR description?